packages/commons/src/router/errorHandlers.ts

```diff
@@ -31,3 +35,35 @@ export function zodiosValidationErrorToApiProblem(
       )
     );
 }
+
+export function errorsToApiProblemsMiddleware(
+  error: SyntaxError,
+  req: WithZodiosContext<express.Request, ExpressContext>,
+  res: Response,
+  _next: NextFunction
+): Response {
```

Comment:
According to [express docs](https://expressjs.com/en/guide/error-handling.html), when you add a custom error handler, you must delegate to the default Express error handler when the headers have already been sent to the client ⬇️ 

```suggestion
): Response {
  if (res.headersSent) {
    return next(err)
  }
```